actionpack/actionpack.gemspec

```diff
@@ -27,7 +27,7 @@ Gem::Specification.new do |s|
   s.add_dependency "activesupport", version
 
   s.add_dependency "rack",      "~> 2.0"
-  s.add_dependency "rack-test", "~> 0.6.3"
+  s.add_dependency "rack-test", "~> 0.7.0"
```

Comment:
Is there any reason to disallow 0.6.3? If not this should be `s.add_dependency "rack-test", ">= 0.6.3"`